synapseclient/core/multithread_download/download_threads.py

```diff
@@ -170,21 +168,16 @@ def _get_pre_signed_info(self) -> PresignedUrlInfo:
         Returns:
             PresignedUrlInfo
         """
-        response = wrap_async_to_sync(
-            coroutine=get_file_handle_for_download(
-                file_handle_id=self.request.file_handle_id,
-                synapse_id=self.request.object_id,
-                entity_type=self.request.object_type,
-                synapse_client=self.client,
-            ),
-            syn=self.client,
+        # noinspection PyProtectedMember
+        response = self.client._getFileHandleDownload(
+            self.request.file_handle_id,
+            self.request.object_id,
+            objectType=self.request.object_type,
         )
         file_name = response["fileHandle"]["fileName"]
         pre_signed_url = response["preSignedURL"]
         return PresignedUrlInfo(
-            file_name=file_name,
-            url=pre_signed_url,
-            expiration_utc=_pre_signed_url_expiration_time(pre_signed_url),
+            file_name, pre_signed_url, _pre_signed_url_expiration_time(pre_signed_url)
```

Comment:
For this I reverted the logic back to it was before I started touching this. Rather than patching the logic I just went back to what it was; Will be removing this in v5 when we remove the deprecated code (This class is deprecated)